manual/modules/resources/partials/typedb-all-latest-links.adoc

```diff
@@ -1,19 +1,19 @@
-| 
```

Comment:
In the table view here: https://typedb.com/docs/manual/install/CE there's a version number we still have to bump... can we make it just say 'latest' there?

Reply:
Believe it's the same on the other ones?

Reply:
No, that's meant to go to github, which doesn't seem to have any `latest` tag like cloudsmith does.

Reply:
I think you can just link it to /releases, which is ordered by release date, which is apprixmately equal to the most recent version @krishnangovindraj